pkg/proxy/jmeter_test.go

```diff
@@ -11,13 +11,14 @@ import (
 )
 
 func TestNewJMeterFromHTTPLoadTest(t *testing.T) {
-	r, err := buildMocFormReq(map[string]string{}, "", string(apisLoadTestV1.LoadTestTypeJMeter))
+	ltType := apisLoadTestV1.LoadTestTypeFake
+	r, err := buildMocFormReq(map[string]string{}, "", string(ltType))
 	if err != nil {
```

Comment:
```suggestion
        require.NoError(t, err)
```

Reply:
I didn't really focus on proxy tests in this PR, we have other tickets to rewrite them. Current changes were from the previous branch (from non private repo).